### DISCUSSION TITLE
Quickstart Bosh Lite - VirtualBox issue

### DISCUSSION THREAD

**Discussion by RicardoPosser** [cloudfoundry/bosh#2464] [Q&A] [answered]
Hi,

I am new to BOSH and I am trying to deploy a VM using the instructions from https://bosh.io/docs/bosh-lite/

I always receive a error about NatNetwork.

I used different machines, versions of VirtualBox, and always get the same message.

Could someone help me ?

Tks in advance,
Ricardo Posser
<img width="1439" alt="PastedGraphic-1" src="https://github.com/cloudfoundry/bosh/assets/132338059/ce6bcaba-66c8-4a9b-b1ac-9c478e64dda2">

**Comment by rkoster** [comment, marked as answer]
There is an open PR that addresses some compatibility issues here: https://github.com/cloudfoundry/bosh-virtualbox-cpi-release/pull/31
